package.json

```diff
@@ -47,6 +47,7 @@
     "eslint-plugin-react": "7.19.0",
     "eslint-plugin-react-hooks": "3.0.0",
     "eslint-plugin-simple-import-sort": "5.0.2",
+    "firebase": "^7.14.0",
```

Comment:
we are pinning exact versions because we don't commit our yarn lockfile so please do that here

Reply:
fixed